app/models/manageiq/providers/nuage/inventory/collector/target_collection.rb

```diff
@@ -210,15 +170,6 @@ def infer_related_ems_refs_api!
         add_simple_target!(:cloud_subnets, subnet['ID'], :options => { :kind => 'L3' })
       end
     end
-
-    references(:network_ports).each do |port|
-      case port['parentType']
-      when 'subnet'
-        add_simple_target!(:cloud_subnets, port['parentID'], :options => { :kind => 'L3' })
```

Comment:
This was also wrong and obviously never used as `port` is in fact a string (ems_ref of a network port) so we would crash when accessing `port['parentID']`. Is not needed anyway now that we handle all those "nuage glitches" if you remember.